mautic/mautic
# Print recent activity logs in code of HTML

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

Many times we need check what was changed in entities.  By default Mautic display recent activity timeline in email, campaigns, segments. But for more details we need go to audit logs and process it manually.
This PR added this output to HTML code as comment.

This should save everybody a tons of time.




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to emails, create and edit it few times (title, subject, from...)
3. Open detail of email, show source and find: **recent activity logs**. You should find printed logs in comments
![image](https://user-images.githubusercontent.com/462477/127009687-54701886-2160-4f3c-92a7-09e02a61950e.png)






<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10295"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
